cost/superseded_instance/superseded_instance.pt

```diff
@@ -229,6 +237,11 @@ script "js_new_costs_request", type: "javascript" do
         {"dimension":"service", "type":"equal", value:"Microsoft.Compute"},
         {"dimension":"resource_type","type":"substring","substring":"Virtual Machines"}
       ]},
+      {"type" : "and", "expressions" : [
+        {"dimension":"vendor","type":"equal","value":"Google"},
```

Comment:
@lawrethao   can you please make sure this is the correct dimension?  is it Google or GCP?